projects/packages/forms/src/contact-form/js/accessible-form.js

```diff
@@ -75,6 +92,16 @@ const initForm = form => {
 
 		clearForm( form, inputListenerMap, opts );
 
+		// First check if any field has a value
+		if ( ! hasAnyFieldValue( form ) ) {
+			setFormError( form, [], {
+				disableLiveRegion: true,
+				message: L10N.atLeastOneField,
+			} );
+			return;
+		}
```

Comment:
Seems like we duplicate this check - here and on line 128 inside isFormValid? Do we need it in both places?

Reply:
thanks for the feedback, we do the check once now!